From 9d33312a2b78960d2cb840f60d2e5d6eb6ac87da Mon Sep 17 00:00:00 2001
From: "Owen W. Taylor" <otaylor@fishsoup.net>
Date: Wed, 10 Feb 2016 17:20:00 -0500
Subject: [PATCH 1/6] examples/cogl-crate.c: fix bug when waiting for next
 frame

The swap_ready variable was never reset, meaning that we weren't
effectively waiting for SYNC events that signal frame completion.
---
 examples/cogl-crate.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/examples/cogl-crate.c b/examples/cogl-crate.c
index 0836b82..1933c7c 100644
--- a/examples/cogl-crate.c
+++ b/examples/cogl-crate.c
@@ -285,6 +285,7 @@ main (int argc, char **argv)
         {
           paint (&data);
           cogl_onscreen_swap_buffers (COGL_ONSCREEN (fb));
+          data.swap_ready = FALSE;
         }
 
       cogl_poll_renderer_get_info (cogl_context_get_renderer (ctx),
-- 
1.8.3.1

